paasta_tools/chronos_serviceinit.py

```diff
@@ -170,7 +170,7 @@ def _format_parents_verbose(job):
     parent_service_instances = [tuple(chronos_tools.decompose_job_id(parent)) for parent in parents]
 
     # find matching parent jobs
-    parent_jobs = [chronos_tools.get_job_for_service_instance(*service_instance)
+    parent_jobs = [chronos_tools.get_job_for_service_instance(*service_instance, include_disabled=False)
```

Comment:
This get_job_for_service_instance is used in the status_chronos_jobs code path only. The other get_job_for_service_instance is in https://github.com/Yelp/paasta/blob/8b83e67b7a8e4073010f35e7bd90c315c7570dfe/paasta_tools/chronos_tools.py#L388, which is used by chronos_rerun, chronos_serviceinit and setup_chronos_job.